dist/angular-google-maps.js

```diff
@@ -6679,13 +6671,15 @@ Original idea from: http://stackoverflow.com/questions/22758950/google-map-drawi
                 map: _gMap
               });
             };
-            if (!_this.validateCoords(scope.center)) {
-              $log.error('angular-google-maps: could not find a valid center property');
+            if (!angular.isDefined(scope.center) && !angular.isDefined(scope.bounds)) {
+              $log.error('angular-google-maps: a center or bounds property is required');
               return;
             }
+            if (!angular.isDefined(scope.center)) {
+              scope.center = new google.maps.LatLngBounds(_this.getCoords(scope.bounds.southwest), _this.getCoords(scope.bounds.northeast)).getCenter();
```

Comment:
You should really be looking and comment on the coffeescript, not the compile js, but the condition on 6674 you mention is !center && !bounds. In other words, if both of them are undefined, log an error. Then, if center is not defined set a center. What exactly are you seeing as a problem with that logic?